src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,159 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int quantity = 0;
+    private Node<T> node;
+    private Node<T> temp;
```

Comment:
Avoid using class Objects. Instead of using Node<T> temp, consider using a more descriptive name for the temporary node.